x-pack/plugin/sql/src/main/java/org/elasticsearch/xpack/sql/optimizer/Optimizer.java

```diff
@@ -1172,6 +1174,9 @@ protected Expression rule(Expression e) {
                     return Literal.of(in, null);
                 }
 
+            } else if (e instanceof NULLIf) {
+                return e; // Special rule SimplifyNullIf is applied instead
+
```

Comment:
Remove this empty line.

Reply:
Was on purpose as we have them in all `else if` blocks.